sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/cryptography_client.hpp

```diff
@@ -85,12 +84,12 @@ namespace Azure {
     /**
      * @brief Initializes a new instance of the #CryptographyClient class.
      *
-     * @param keyId The key identifier of the #KeyVaultKey which will be used for cryptographic
-     * operations.
-     * @param credential A #TokenCredential used to authenticate requests to the vault, like
-     * DefaultAzureCredential.
-     * @param options #CryptographyClientOptions for the #CryptographyClient for local or remote
-     * operations on Key Vault.
+     * @param keyId The key identifier of the #Azure::Security::KeyVault::Keys::KeyVaultKey which
+     * will be used for cryptographic operations.
+     * @param credential An Azure::Core::Credentials::TokenCredential used to authenticate requests
```

Comment:
Should this have an anchor tag?

Reply:
Yes, but anchor tags can't span packages :(.